webapi/src/main/scala/org/knora/webapi/slice/admin/repo/rdf/RdfConversions.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © 2021 - 2024 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.slice.admin.repo.rdf
+
+import dsp.valueobjects.V2
+import org.knora.webapi.slice.admin.domain.model.KnoraProject.*
+import org.knora.webapi.slice.common.repo.rdf.LangString
+
+object RdfConversions {
+  implicit val shortcodeConverter: String => Either[String, Shortcode] =
+    Shortcode.from
+
+  implicit val shortnameConverter: String => Either[String, Shortname] =
+    Shortname.from
+
+  implicit val longnameConverter: String => Either[String, Longname] =
+    Longname.from
+
+  implicit val descriptionConverter: LangString => Either[String, Description] = langString =>
+    Description.from(V2.StringLiteralV2(langString.value, langString.lang))
+
+  implicit val keywordConverter: String => Either[String, Keyword] =
+    Keyword.from
+
+  implicit val logoConverter: String => Either[String, Logo] =
+    Logo.from
+
+  implicit val statusConverter: Boolean => Either[String, Status] =
+    value => Right(Status.from(value))
+
+  implicit val selfjoinConverter: Boolean => Either[String, SelfJoin] =
+    value => Right(SelfJoin.from(value))
+}
```

Comment:
Isn't this the place where our DB inconsistency could blow up? I mean the recently updated value objects rules.

Reply:
I'll check, thanks for reminding me

Reply:
checked with the DB: all seems ok